bigframes/ml/llm.py

```diff
@@ -244,7 +244,7 @@ def predict(
 
         Args:
             X (bigframes.dataframe.DataFrame or bigframes.series.Series):
-                Input DataFrame or Series, which contains only one column of prompts.
+                Input DataFrame or Series, can contain one or more columns. If multiple columns in the DataFrame, it must contain a "prompt" column for prediction.
```

Comment:
nit: "If multiple columns are in the DataFrame, they must ..." and for other docs too

Reply:
"it" refers to the DataFrame. Can add "are" in "If multiple columns are in the DataFrame"

Reply:
done